### PR TITLE
Add 3 new levels, update readme

### PR DESCRIPTION
**Pull request by argallegos**
This PR adds three new scenes to the Cesium for Unreal Samples. Since the last scene is about material editing, it also includes four example materials to help users get started, including template materials based on the default tileset materials.

The [readme](https://github.com/CesiumGS/cesium-unreal-samples/blob/add-new1.8-levels/README.md) has also been updated. I added information about the new levels, linked sample levels to existing tutorials, and reworded some of it.

These levels are still using the December token - After this is merged, we'll need to update all the tokens in the levels before updating it on the Marketplace.

New levels and their text:

- 07_CesiumMultiView
   - In this scene, explore the world from multiple views. Using  a Scene Capture 2D and a Render Target, you can visualize multiple different areas of a tileset at the same time.
Press "Play" to test out the scene. You'll see a minimap over the player controller, and a view from space.
 Press "1" to return to this view.
- 08_CesiumClipping (3 info boards)
   - In this level, you'll explore the Cesium Cartographic Polygon actor, which lets you cut out sections of tilesets. This can be used to combine data in to a cohesive scene, and replace elements with higher-fidelity objects.
 In the World Outliner, hide the Aerometrex Denver tileset by clicking the eyeball icon. You'll see that Cesium World Terrain has been hidden to make room for the photogrammetry.
Press "2" in the editor to take a closer look.
Press "1" in the editor to return to this view.
   - By using the Cartographic Polygon actor, you can line up different datasets without worrying about one showing through the other. This is a material effect, and it has no effect on collision. Clipping is not recommended for precise physics-based applications.
   Click on the Cesium World Terrain actor in the World Outliner to see how this is set up. The CesiumCartographicPolygon is connected to the tileset through the tileset's CesiumPolygonRasterOverlay component. You can clip multiple tilesets with a single Cartographic Polygon. Here, OSM buildings uses the same polygon as Cesium World Terrain for clipping.
Press "3" in the editor to keep exploring.
Press "1" in the editor to return to the start.
   - Here, a CartographicPolygon actor has been used on the terrain to cut out this lake. A landscape has been used beneath to form the lakebed and a water body has been added to create the water surface. 
You can add more points to a Cartographic Polygon by selecting a point and pressing the "alt" key while clicking and dragging that point.
Press "1" in the editor to return to the start.
- 09_CesiumMaterialEditing
   - Welcome to Crater Lake, Oregon. In this level, you'll learn how to use different materials on Cesium 3D Tilesets.
Crater Lake, like many other bodies of water, is not currently shown on Cesium World Terrain's watermask. However, you can still make it look like water by using Cesium Cartographic Polygons and an edited material.  
Click on Cesium World Terrain. You'll find the tileset's material in the Details panel, under Cesium|Rendering. Try removing MI_CesiumCustomWater from the tileset to see how this scene would look with the default material. 
In CesiumSamples/Materials, you'll find some other example materials. Try replacing the tileset's material with MI_CesiumElevationBands, MI_CesiumGridLines, or MI_HeightBlend.
Want to make your own materials? There is a template material and functions in the Materials/Templates folder. You can use this as a base for your own creations.
 Press "1" to return to this view.

Please feel free to suggest revisions to the text. 

Here are the example materials included with the last level:
All materials are commented and organized to make things as easy as possible for users.

MI_CesiumCustomWater
image
This material allows users to use Cartographic Polygons to draw out custom water areas using the `CustomWater` Material Layer Key. They can also draw out areas of land using the `CustomLand` material layer key.

MI_CesiumElevationBands
image
Adds elevation bands over raster overlays. Users can define the width of the band in meters, as well as the line width and the color.

MI_CesiumGridLines
image
Adds grid lines over raster overlays. Users can define the size of the grid in meters and the width of the grid lines.

MI_CesiumHeightBlend
image
Colors the terrain based on the Z height. Users can define the blend blur, blend offset, and colors to blend between. Users can also define how opaque the blend effect is.

Also included are some templates for the material, material instance, material layer, and material layer blend. These will be used in the material editing tutorial and material editing assistance given to users on the forum. Each template is commented clearly to help users understand how the parts work together.
Here's an example of the comments, in this case on the base template material.
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,55 +2,84 @@
...
 ### :rocket: Get Started
 
 **[Download Cesium for Unreal Samples from Unreal Engine Marketplace](https://cesium.com/unreal-samples-marketplace?utm_source=cesium-unreal-samples&utm_medium=github&utm_campaign=unreal)**
 
-You will also need to **[download Cesium for Unreal plugin from Unreal Engine Marketplace](https://cesium.com/unreal-marketplace?utm_source=cesium-unreal&utm_medium=github&utm_campaign=unreal)**
+You will also need to download the [Cesium for Unreal plugin from the Unreal Engine Marketplace](https://cesium.com/unreal-marketplace?utm_source=cesium-unreal&utm_medium=github&utm_campaign=unreal).
+
+The project comes pre-populated with temporary [access tokens](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/#step-1-create-an-account-and-get-a-token) to allow you to view the datasets. These tokens refresh every month, so be sure to replace the token on every tileset with a token from your own Cesium ion account before building off of these levels.
+
+If all the tilesets suddenly disappear from your levels, you may have been using a sample token. Try replacing them with tokens from your account.
```
Comment (shehzan10) [inline README.md:20]: > The project comes pre-populated with temporary [access tokens](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/#step-1-create-an-account-and-get-a-token) to allow you to view the datasets. These tokens refresh every month, so be sure to replace the token on every tileset with a token from your own Cesium ion account before building off of these levels.
> 
> If all the tilesets suddenly disappear from your levels, you may have been using a sample token. Try replacing them with tokens from your account.

Good to have this documented, but not in the readme.

File: README.md [hunk 2]
```diff
@@ -2,55 +2,84 @@
...
+
+This level has lighting, fog, volumetric clouds, and a post processing volume to create a picturesque scene at Mt. Fuji. You should try tweaking the setting to customize the scene and create different effects. Not sure where to start? Check out the [Lighting and Rendering tutorial](https://cesium.com/learn/unreal/unreal-post-processing/). We would love to see what you create!
+
+### :six: Level 6 - Metadata
+
+Cesium for Unreal enables you to access metadata encoded in your 3D Tiles datasets. In this level, explore the buildings in New York City and see what information is included in the tileset about each building.
+
+Looking to use metadata in your project? Check out the [Visualizing Metadata tutorial](https://cesium.com/learn/unreal/unreal-visualize-metadata/) to learn more.
+
+### :seven: Level 7 - Multiple Views
```
Comment (shehzan10) [inline README.md:67]: Lets add a screenshot for this.

**Review by shehzan10 — COMMENTED** [review]
@argallegos I think the readme updates are great. We're probably right on the cusp of how many levels we want in the readme, so we should discuss the structure for future additions.

**Review by nithinp7 — COMMENTED** [review]
Great work @argallegos, this looks good to me! Feel free to merge it once you've addressed Shehzan's suggestions.

**Review by nithinp7 — APPROVED** [review]

**Pull request merged**